Update readme for new docs (#147)
GRTLCollaboration/GRTeclyn: README.md

# GRTeclyn

> GRTeclyn (previously referred to as GRAMReX) is a new numerical relativity code developed by the [GRTL Collaboration](www.grtlcollaboration.org) that is currently still under development.  It is a port of the [GRChombo code](https://github.com/GRChombo/GRChombo) (based on the Chombo libraries) to the [AMReX](https://amrex-codes.github.io/) library in order to leverage AMReX's support for GPUs and ongoing active development.

Reviewer: For some reason the GRTL Collaboration link doesn't work for me (GitHub thinks it's a relative link)
```suggestion
GRTeclyn (previously referred to as GRAMReX) is a new numerical relativity code developed by the [GRTL Collaboration](https://www.grtlcollaboration.org) that is currently still under development.  It is a port of the [GRChombo code](https://github.com/GRChombo/GRChombo) (based on the Chombo libraries) to the [AMReX](https://amrex-codes.github.io/) library in order to leverage AMReX's support for GPUs and ongoing active development.
```

# GRTeclyn

GRTeclyn (previously referred to as GRAMReX) is a new numerical relativity code developed by the [GRTL Collaboration](www.grtlcollaboration.org) that is currently still under development.  It is a port of the [GRChombo code](https://github.com/GRChombo/GRChombo) (based on the Chombo libraries) to the [AMReX](https://amrex-codes.github.io/) library in order to leverage AMReX's support for GPUs and ongoing active development.

The AMReX documentation can be found [here](https://amrex-codes.github.io/amrex/docs_html).

The name follows a similar pattern to GRChombo, namely "GR" + "\<Tool in a foreign language\>". In this case, "Teclyn" is a Welsh word for "Tool".

> ### Development status

Reviewer: ```suggestion
## Development status
```

The current timeline for releasing GRTeclyn is December 2025 - the goal is that at this point it should contain well tested methods for vacuum binaries with gravitational wave extraction, and capabilities to add scalar fields. Further features will be ported later.

> A detailed list of the status of particular features can be found in the [documentation](https://grtlcollaboration.github.io/GRTeclyn/#summary-of-features).

Reviewer: I would prefer not to put a date in our README. Unless we intend to use GitHub "releases" (or tags) regularly, we should avoid using the term "release". Let's just refer to the page in our documentation.
```suggestion
Please consult this [documentation page](https://grtlcollaboration.github.io/GRTeclyn/#summary-of-features) for a list of the development status of specific features.
```

Author: Oh ok... I knew you wouldn't like this :-)

Reviewer: Why did you include it in the first place then?

Author: I was hoping you'd be too busy to notice it 😆

> ### Documentation

Reviewer: ```suggestion
## Documentation
```

> The documentation is also under construction, but can be found [here](https://grtlcollaboration.github.io/GRTeclyn/). Note that the wiki in GitHub is no longer in use.

Reviewer: ```suggestion
Documentation can be found [here](https://grtlcollaboration.github.io/GRTeclyn/) (under construction). Note that the GitHub wiki is no longer in use.
```
